### DISCUSSION TITLE
Install via Homebrew?

### DISCUSSION THREAD

**Discussion by DitchComfort** [LunarVim/LunarVim#3578] [Q&A]
Do latest LunarVim work properly if you have neovim installed via Homebrew? I know people are recommending building from source but...

**Comment by mamachanko** [comment]
It worked for me. But that's no proof. 😂

**Reply by DitchComfort** [reply]
Okey. And everything installed properly? Nothing missing or error messages?

**Reply by mamachanko** [reply]
Yes. It works as I expect it to.
